edgelet/contrib/snap/command-chain/handle-exit-status-153.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eu
+EXIT_STATUS=0
+
+"$@" || EXIT_STATUS=$?
+
+if [ $EXIT_STATUS -eq 153 ] ; then
+    snapctl stop $SNAP_INSTANCE_NAME.aziot-edged --disable
```

Comment:
Does the `--disable` argument here mean that `--enable` is required whenever the subsequent `snapctl start/restart` is called?

Reply:
Yes, this would prevent snapd from attempting to start it on `snapctl start`, `snap start`, system reboot, snap upgrade, etc. So either I need to add `--enable` in the `configure` hook, or I should just drop the `--disable` which will mean that the service will try again once to start up on system reboot, snap upgrade, etc, but won't thrash in those scenarios.

Reply:
Ok, I think starting occasionally, but not thrashing, is fine. So whichever approach you think is best.

Reply:
@damonbarry went with the "starting occasionally" approach